src/content/grant/2021_Quantum_algorithms.md

```diff
@@ -5,8 +5,9 @@ month: 3
 day: 10
 country: SG
 tags:
-  - quantum education
-  - quantum algorithm
+  - education
+  - quantum algorithms
```

Comment:
In some places `quantum algorithms` is used and in other it's just `algorithms`. I think we should pick one for consistency.